src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl

```diff
@@ -194,11 +194,14 @@ function HydrostaticFreeSurfaceModel(; grid,
     implicit_solver   = implicit_diffusion_solver(time_discretization(closure), grid)
     prognostic_fields = hydrostatic_prognostic_fields(velocities, free_surface, tracers)
 
-    timestepper = TimeStepper(timestepper, grid, tracernames(tracers);
+    # QuasiAdamsBashforth2 requires χ as a third positional argument, while SplitRungeKutta3 requires
+    # the prognostic_fields as a third positional argument.
+    timestepper_arg = timestepper == :QuasiAdamsBashforth2 ? 0.1 : prognostic_fields
```

Comment:
no no no no

Reply:
make the constructors the same. prognostic_fields as third argument, some scalar parameter as 4th.